index.html

```diff
@@ -0,0 +1,203 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script defer src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div class="topbar">
+        <a href="https://github.com/vangelif"
+          ><i class="fa-brands fa-github"></i
+        ></a>
+        <a href="https://www.linkedin.com/in/vangfot/"
+          ><i class="fa-brands fa-linkedin"></i
+        ></a>
+        <a href="https://twitter.com/vangfot"
+          ><i class="fa-brands fa-twitter"></i
+        ></a>
+        <a href="#">Logout</a>
+        <a href="#">Contact</a>
+        <a href="https://vangelis.website/essays">Blog</a>
+      </div>
+      <nav class="navigation">
+        <div>
+          <a id="logo" href="index.html"><i class="fas fa-bullseye"></i></a>
+          <ul class="desktopnav">
+            <a id="activelink" href="./index.html">
+              <li>Home</li>
+            </a>
+            <a href="./about.html">
+              <li>About</li>
+            </a>
+            <a href="">
+              <li>Sponsor</li>
+            </a>
+            <a href="">
+              <li>News</li>
+            </a>
+            <a href=""><button type="button">Join Now</button></a>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <div class="" id="mobile-menu">
+      <div id="close">
+        <i class="fa-solid fa-xmark"></i>
+      </div>
+      <ul>
+        <li><a id="homelink" href="index.html"> Home</a></li>
+        <li><a id="aboutlink" href="about.html"> About</a></li>
+        <li><a id="sponsorlink" href="#"> Sponsor</a></li>
+        <li><a id="newslink" href="#"> News</a></li>
+        <li><a id="join-us" href="#"> Join Us</a></li>
+      </ul>
+    </div>
+    <!-- intro -->
+    <section class="intro">
+      <div class="intro-top">
+        <button id="hamburgerBtn" href="">
+          <i class="fa-solid fa-bars"></i>
+        </button>
+        <p>"Hello! Sharing World"</p>
+        <h1>
+          SINOLOGY COMMONS<br />
+          GLOBAL SUMMIT<br />
+          2015
+        </h1>
+      </div>
+      <p class="intro-mid">
+        A joyful Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla
+        architecto omnis vitae deleniti nostrum necessitatibus? Hic sunt minus
+        quos impedit modi nam nemo?
+      </p>
+
+      <div class="intro-bottom">
+        <h3><strong>2023.02.06(MON) ~ 09(THU)</strong></h3>
+        <aside>@ National Museum of China, Art Center Nabi and more</aside>
+      </div>
+    </section>
+    <!-- main program -->
+    <section class="program">
+      <h3>Main Program</h3>
+      <hr />
+      <ul>
+        <li>
+          <span><i class="fas fa-robot"></i></span>
+          <h3>Lecture</h3>
+          <p>
+            Listen to the speakers Lorem, ipsum dolor. Laboriosam nemo rerum
+            quod praesentium eni.
+          </p>
+        </li>
+        <li>
```

Comment:
- [ ] Kindly ensure that the title of your `program cards` are on the same level as in the original design and add `hover effect` to them.

![image](https://user-images.githubusercontent.com/98453979/218110814-f8af064e-ffee-495b-a420-a80d01eb52c1.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/218110937-972163ed-f633-4a9f-b338-8d8280e8c4d1.png)